doc/mod_wsgi.md

```diff
@@ -0,0 +1,200 @@
+# Running SATOSA Using Apache HTTP Server and mod\_wsgi
+
+This document describes how to deploy and run the SATOSA proxy using
+Apache HTTP Server and mod\_wsgi. To be concrete this document details
+deploying SATOSA on the latest CentOS 7.x release.
+
+## Dependencies
+
+```
+yum install epel-release
+yum install httpd mod_ssl httpd-devel python34 python34-devel
+yum install xmlsec1-openssl gcc curl
+```
+
+Install the latest production release of pip and use it to install the latest 
+production release of mod\_wsgi:
+
+```
+curl https://bootstrap.pypa.io/get-pip.py | python3
+pip install mod_wsgi
```

Comment:
The mod_wsgi version packaged by RedHat and available with CentOS, even that from the SCL repositories, is old and buggy and explicitly disavowed by the package's developer.

Reply:
If things are that bad with CentOs packaging, maybe one shouldn't be using CentOs in the first place. This is not a pun, I really mean it. If the distribution cannot provide you with up-to-date packages or backported security fixes, then it provides no value. 
I am not aware of CentOs development processes, but if that's the situation I would suggest to abandon CentOs.

Reply:
When I have a choice I do not choose RHEL based operating systems, but it is not my choice. But it is a choice that many people do make. For this example I documented CentOS because that is the platform we are using for the project that paid for the work and the time to make contributions to SATOSA.